#! /bin/sh /usr/share/dpatch/dpatch-run

## 13_vdr_1.3.45_sources.conf-fix.dpatch by Darren Salt <linux@youmustbejoking.demon.co.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes sources.conf of vdr 1.3.45

@DPATCH@
diff -urNad vdr-1.3.45/sources.conf /tmp/dpep.1E57JD/vdr-1.3.45/sources.conf
--- vdr-1.3.45/sources.conf	2006-03-19 14:03:46.000000000 +0100
+++ /tmp/dpep.1E57JD/vdr-1.3.45/sources.conf	2006-03-26 20:58:58.119450500 +0200
@@ -165,7 +165,7 @@
 S105W   AMC15
 S107.3W Anik F1/F1R
 S110W   DirecTV 5 & Echostar 6/8
-A111.1W Anik F2
+S111.1W Anik F2
 S113W   Solidaridad 2
 S119W   Echostar 7 & DirecTV 7S
 S121W   Echostar 9 & Intelsat Americas 13
